engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/callcaching/CallCacheHashingJobActor.scala

```diff
@@ -137,7 +141,9 @@ class CallCacheHashingJobActor(jobDescriptor: BackendJobDescriptor,
     val hashingJobActorData = CallCacheHashingJobActorData(fileHashRequests.toList, callCacheReadingJobActor)
     startWith(WaitingForHashFileRequest, hashingJobActorData)
 
-    val initialHashingResult = InitialHashingResult(initialHashes, calculateHashAggregation(initialHashes, MessageDigest.getInstance("MD5")))
+    val aggregatedBaseHash = calculateHashAggregation(initialHashes, MessageDigest.getInstance("MD5"))
+    val executionBucketHint = initializationData collect { case d: StandardInitializationData => ExecutionBucketHint(d.workflowPaths.executionRootString) } toList
```

Comment:
This is not limited to gcs buckets is it ? Won't it also filter on SFS ?

Reply:
Yes which is why it fails for non-GCS, I need to loop back and fix this particularly POCy bit. :)